src/lib/common/tag.h

```diff
@@ -30,6 +30,22 @@
 #include "common/Format.h"
 
 
+// Macros to be used for JSON rendering
+
+#define JSON_PROP(name) std::string("\"" + std::string(name) + "\":")
```

Comment:
but then we may have problems concatenating the macro result with other strings. I have suffered it :(